Makefile.toml

```diff
@@ -102,6 +102,13 @@ alias = "lqs-tests-flow"
 ################
 # Custom tasks #
 
+[tasks.check-all-format]
```

Comment:
:thinking: Not sure if we actually need this task?
The aggregation (i.e. calling both) could also just happen in the `pre-commit` file. I don't see a usecase where you would call both of them manually.

Would make our Makefile simpler.